src/player/player_update.cpp

```diff
@@ -616,27 +619,37 @@ void UpdatePlayer()
                     else
                     {
                         if(Player[A].Controls.Left)
+                        {
                             if(!Player[A].Controls.Run)
                                 Player[A].Location.SpeedX = -1.5;
                             else
                                 Player[A].Location.SpeedX = -3;
+                        }
+                        
```

Comment:
before `else` - line not needed,
but if `if` without `else`, then line is needed.